server/cccp_reader.py

```diff
@@ -24,20 +24,20 @@ def main():
     if(key == "test-skip"):
       test_skip = cccp_yml["test-skip"]
 
-    if(key == "build-script"):
+    if(key == "build-script" and cccp_yml["build-script"]):
```

Comment:
this is raise exception of key not found in cccp_yml, if it is not defined and stop the loop. may be we should put this in try catch block? similar for the below keys also.